### PR TITLE
add Transcript Loader for Bilibili

### PR DESCRIPTION
**Pull request by AlexZhangji**
This loader utilizes the `bilibili_api` to fetch the text transcript from [Bilibili](https://www.bilibili.com/), one of the most beloved long-form video sites in China.

With this, users can easily obtain the transcript and general infor from the site. 

```
# Sample results for one video:

"""Video Title: {video_title},
Description: {description},
Transcript:  {transcript}"""
```

### CODE REVIEW

File: loader_hub/bilibili/base.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+"""Simple Reader that reads transcript and general infor of Bilibili video."""
+from typing import Any, List, Optional
+import warnings
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class BilibiliTranscriptReader(BaseReader):
+    """Bilibili Transcript and video infor reader."""
```
Comment (emptycrown) [inline loader_hub/bilibili/base.py:10]: Typo I assume. infor -> info

File: loader_hub/bilibili/base.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+"""Simple Reader that reads transcript and general infor of Bilibili video."""
+from typing import Any, List, Optional
+import warnings
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class BilibiliTranscriptReader(BaseReader):
```
Comment (emptycrown) [inline loader_hub/bilibili/base.py:9]: You'll need to add this to the `library.json` file for it to be picked up by LlamaIndex's `download_loader` and the LlamaHub website.

**Review by emptycrown — CHANGES_REQUESTED** [review]
Thank you! Look good overall -- left a couple of comments.

File: loader_hub/bilibili/base.py (hunk 2 above)
Comment (AlexZhangji) [reply loader_hub/bilibili/base.py:9]: thanks for the reply! just added.

File: loader_hub/bilibili/base.py (hunk 1 above)
Comment (AlexZhangji) [reply loader_hub/bilibili/base.py]: thanks Jesse, fixed.

**Review by emptycrown — APPROVED** [review]

**Pull request merged**